packages/ui/components/form-core/test-suites/FormRegistrationMixins.suite.js

```diff
@@ -256,6 +257,23 @@ export const runRegistrationSuite = customConfig => {
       ]);
     });
 
+    describe('FormRegisteringMixin', () => {
+      it('dispatches the form-element-register event with correct details upon connectedCallback', async () => {
+        const el = /** @type {RegistrarClass} */ (
+          await fixture(html`
+            <${parentTag} .formElementRegisterComposed="${true}">
+              <${childTag}></${childTag}>
+            </${parentTag}>
+          `)
+        );
+        const eventSpy = sinon.spy();
+        el.addEventListener('form-element-register', eventSpy);
+        el.connectedCallback();
```

Comment:
this should not be needed
```suggestion
```

Reply:
The test breaks without calling `el.connectedCallback()` explicitly